### DISCUSSION TITLE
百度网盘下载的时候，出现异常，如图，不知道怎么形容！

### DISCUSSION THREAD

**Discussion by heihei999** [AlistGo/alist#3371] [Q&A]
![QQ截图20230212224753](https://user-images.githubusercontent.com/64182111/218318062-bd1050a2-8dc6-4d0d-b49d-a8598754406a.png)

![218317915-95a0be96-cbad-46a3-9731-da287cf6669c](https://user-images.githubusercontent.com/64182111/218318195-3bf74afd-ad16-4ebf-a927-b2d5c0d79394.png)
情况和设置如图，alist在本地，还没有配置Aria2，本人是小白，比较菜，麻烦大佬帮忙解答！非常感谢

**Comment by anwen-anyi** [comment]
建议你去看一下文档非官方界面哪里的描述~可能是你IDM的问题导致日志没有返回 给你返回了一个 “假”文件

**Reply by heihei999** [reply]
解决了，添加useragent，可以解决下载100M内的文件，但是大文件还是不行，用非官方的还是下载不了

**Reply by anwen-anyi** [reply]
> 解决了，添加useragent，可以解决下载100M内的文件，但是大文件还是不行，用非官方的还是下载不了

没有发现非官方和谐了么........QAQ
